voxconf/2025.md

```diff
@@ -68,15 +68,27 @@ Connect with us on our Slack in #voxconf2025 and/or on IRC in #voxpupuli-voxconf
 
 {: .table .table-striped .table-hover .w-75}
```

Comment:
with the 75char limit we now have a line break. Is that intentional or should the table become wider?

<img width="2073" height="941" alt="2025-07-25_11-26" src="https://github.com/user-attachments/assets/f337c9c6-715b-404c-8bcc-557ab43500ab" />

Reply:
oh, not intentional. will check this again.

for next voxconf: enforce shorter titles 😆

Reply:
i would say we drop the 75%-rule because the tables is so wide already and if we add also the recording links to each talk it will be even longer.